### PR TITLE
Comments and cleanup

### PR DESCRIPTION
**Pull request by ajnonaka**
-Clean up Make.package
-remove unused temp *~ files
-remove redundant files

### CODE REVIEW

File: Source/main.cpp [hunk 1]
```diff
@@ -75,7 +73,7 @@ struct amress_solver
       int Nghost = 2; // 2nd order accuracy scheme is used for convective terms
 
       // Ncomp = number of components for each array
-      // The userCtx has 2 components: phi and pressure
+      // The userCtx has 2 components: pressure and phi
```
Comment (ajnonaka) [inline Source/main.cpp:76]: phi is initialized to zero but then never used again - what is phi for?

File: Source/main.cpp [hunk 2]
```diff
@@ -700,7 +698,7 @@ void main_main ()
 	// Copy the intermediate values to the next sub-iteration
         for ( int comp=0; comp < AMREX_SPACEDIM; ++comp)
         {
-            MultiFab::Copy(velImPrev[comp], velCont[comp], 0, 0, 1, 0);
+            MultiFab::Copy(  velImRK[comp],     velCont[comp], 0, 0, 1, 0);
```
Comment (ajnonaka) [inline Source/main.cpp:701]: I simplified the code by copying the initial condition into velImRK here, and the at the beginning of each iteration copying velImRK into velIMPrev (and skipping the copy at the end).  I verified the solution evolves identically.

File: Source/main.cpp [hunk 3]
```diff
@@ -681,7 +679,7 @@ void main_main ()
         userCtx.FillBoundary(geom.periodicity());
 
 	// Enforce the physical boundary conditions
-        enforce_boundary_conditions(velCart, type1, Nghost, bc_lo, bc_hi, n_cell);
+        enforce_boundary_conditions(userCtx, type1, Nghost, bc_lo, bc_hi, n_cell);
```
Comment (ajnonaka) [inline Source/main.cpp:682]: I think this was a bug (undetectable until walls are turned on)

File: Source/main.cpp [hunk 4]
```diff
@@ -525,7 +523,7 @@ void main_main ()
      */            
 
     //------------------------------------------------------
-    // 1 of 4 options: 'pressure', 'phi', 'velocity', 'flux'
+    // 1 of 4 options: 'pressure', 'velocity', 'flux', 'velocity'
```
Comment (ajnonaka) [inline Source/main.cpp:526]: Might as well combine type2 and type4, they are the same.

File: Source/main.cpp [hunk 5]
```diff
@@ -795,6 +786,7 @@ void main_main ()
         Poisson_Update_Solution (phi_solution, grad_phi, userCtx, velCont, velImRK, geom, ba, dm, bc, dt);
 
 	// Update velCart from the velCont solutions
+        // SHOULD THIS BE velCont instead of velImRK?
```
Comment (ajnonaka) [inline Source/main.cpp:789]: Is this a bug?

**Review by trunglendsu — APPROVED** [review]

File: Source/main.cpp (hunk 5 above)
Comment (trunglendsu) [reply Source/main.cpp:789]: Yes, this is the bug

File: Source/Make.package
Comment (trunglendsu) [inline Source/Make.package:1]: Yes, this is a nice way of putting those files separately

**Pull request merged**